operator/api/v1alpha1/keptnapp_types.go

```diff
@@ -73,3 +76,32 @@ func init() {
 func (w KeptnApp) GetAppVersionName() string {
 	return strings.ToLower(w.Name + "-" + w.Spec.Version)
 }
+
+func (v KeptnApp) SetSpanAttributes(span trace.Span) {
+	attributes := v.GetSpanAttributes()
+	for _, attribute := range attributes {
+		span.SetAttributes(attribute)
+	}
```

Comment:
you can pass the array directly: 
```suggestion
	span.SetAttributes(attribute...)
```